music-player/src/components/SongsComponent.vue

```diff
@@ -2,6 +2,15 @@
   <p>
     Current playlist: <strong>{{ selectedPlaylist.name }}</strong>
   </p>
+
+  <ol v-if="selectedPlaylist.SongsList.length !== 0">
+    <li v-for="song in selectedPlaylist.SongsList" :key="song.title">
+      <p>{{ song.title }} - {{ song.artist }} ({{ song.year }})</p>
+    </li>
+  </ol>
+  <ul v-else>
+    <li>This playlist is empty</li>
```

Comment:
NICE!!!

Reply:
Just would add some more descriptive message to the user like "Your playlist is empty. Click 'Add song' to start adding songs to Your playlist" or something like that